File: kite-types/manifest/manifest.go
package manifest

import "github.com/merlinfuchs/kite/kite-types/event"

type Manifest struct {
	Events          []event.EventType `json:"events"`
	DiscordCommands []DiscordCommand  `json:"discord_commands"`
	ConfigSchema    ConfigSchema      `json:"config_schema"`
}

File: kite-sdk-rust/crates/kiters-sys/src/manifest.rs
use serde::Serialize;

#[derive(Debug, Default, Serialize)]
pub struct Manifest {
    pub events: Vec<String>,
    pub discord_commands: Vec<DiscordCommand>,
    pub config_schema: ConfigSchema,
}

#[derive(Debug, Serialize)]
pub struct DiscordCommand {}

#[derive(Debug, Default, Serialize)]
pub struct ConfigSchema {}
